### ISSUE TITLE
Please make the terraform install location configurable

### ISSUE THREAD

**Issue by kevwil** [tfutils/tfenv#66]
As homebrew installs new versions of tfenv and cleans up the old versions, all existing terraform installations created by tfenv get deleted because they were installed under tfenv itself. https://github.com/Homebrew/homebrew-core/issues/16077#issuecomment-322051639

**Comment by kevwil** [comment]
Why was this concern not addressed in the 0.6.0 release?

**Comment by kamatama41** [comment]
@kevwil 
Sorry for late reply. I didn't have enough time to maintain and enhance tfenv so far. So I'd appreciate if you create a PR to do that 🙏

**Comment by espoelstra** [comment]
@kevwil if you use brew to install from the master branch via `HEAD` and upgrade when new commits occur do you still lose the terraform versions?

**Comment by stevemossovo** [comment]
Still experiencing this issue in `tfenv` `2.0.0`:

```
$ terraform get -update
cat: /usr/local/Cellar/tfenv/2.0.0/version: No such file or directory
Version could not be resolved (set by /usr/local/Cellar/tfenv/2.0.0/version or tfenv use <version>)
```

**Comment by Zordrak** [comment]
I'm not really following this one.. and want to clean up as this is very old.
I believe the initial issue is now covered by #127

I think the most recent comment is a misunderstanding of the need to use after install due to #167 ?
If not, and a bug is present, please open a new issue clearly aimed at the bug.
